src/afl-forkserver.c

```diff
@@ -839,8 +894,6 @@ fsrv_run_result_t afl_fsrv_run_target(afl_forkserver_t *fsrv, u32 timeout,
 
   MEM_BARRIER();
 
-  if (fsrv->shdmem_fuzz_len) write_value += (fsrv->shdmem_fuzz_len << 8);
```

Comment:
Don't really understand this change. Is it not needed anymore?

Reply:
no, the length is now transfered via the shm fuzz map. more flexible